### ISSUE TITLE
Disabling CookieRedirector throws Exception

### ISSUE THREAD

**Issue by GALPR** [dachcom-digital/pimcore-i18n#149]
| Q                | A      |
|------------------|--------|
| Bug report?      | yes|
| Feature request? | no |
| BC Break report? | no |
| RFC?             | no |

An exception is thrown when disabling the CookieRedirector service via the following config:
```yaml
i18n:
  registry:
    redirector:
      cookie:
        enabled: false
```

> "cookie" redirector identifier does not exist

The line that causes this issue is https://github.com/dachcom-digital/pimcore-i18n/blob/fac7a85653a8771203dab56cef8385bc0794addd/src/EventListener/DetectorListener.php#L104

I think this statement could simply be wrapped in `if ($this->redirectorRegistry->has('cookie'))`.

**Comment by solverat** [comment]
This is a regression of https://github.com/dachcom-digital/pimcore-i18n/pull/147

**Comment by solverat** [comment]
fixed in 5.0.1
